recipes/r-rgeos/meta.yaml

```diff
@@ -0,0 +1,59 @@
+{% set version = '0.3-26' %}
+
+{% set posix = 'm2-' if win else '' %}
+{% set native = 'm2w64-' if win else '' %}
+
+package:
+  name: r-rgeos
+  version: {{ version|replace("-", "_") }}
+
+source:
+  fn: rgeos_{{ version }}.tar.gz
+  url:
+    - https://cran.r-project.org/src/contrib/rgeos_{{ version }}.tar.gz
+    - https://cran.r-project.org/src/contrib/Archive/rgeos/rgeos_{{ version }}.tar.gz
+  sha256: 98524a0b8113abe6c3d0ecc1f2f66e7ab6d40c783a76158cfc017e1ab1e3f433
+
+build:
+  number: 0
+  skip: true  # [win32]
+
+  rpaths:
+    - lib/R/lib/
+    - lib/
+
+requirements:
```

Comment:
Did you consider adding `geos` as a dependency? The [CRAN entry](https://cran.r-project.org/package=rgeos) has the following field:

> SystemRequirements: GEOS (>= 3.2.0); for building from source: GEOS from http://trac.osgeo.org/geos/; GEOS OSX frameworks built by William Kyngesburye at http://www.kyngchaos.com/ may be used for source installs on OSX.

Reply:
Done!